workers/unity/Assets/Playground/Config/MetricConfig.cs

```diff
@@ -0,0 +1,12 @@
+namespace Playground
+{
+    public delegate float CalculateLoadDelegate(float fps);
+
+    public static class MetricConfig
```

Comment:
I'd rather we chose a different approach than global statics, if we can help it.
More fundamental than that, however - why are we including this at all? Why isn't the default calculation built into the system enough? Where's the use case?

Reply:
W.r.t. global statics: same pattern used is currently used in `PlayerLifecycleConfig`. This is likely to be changed.

Reply:
I don't think there is any built in calculation. Either on the sdk end, on how often to send, or from unity end, on the current fps (which always surprises me).

The use case is the rare event is to check when running in the cloud if the worker is overloaded. It came up a fair bit in the last game jam.

Reply:
Deleted my comment as I'd completely misunderstood the comment I was responding to.

Reply:
+1 to no global statistics. This a system specific set of constants to do a specific job. If anything we just need some constants for the sake of not having magic numbers.

Reply:
Given that the entire system exists in user code - I think its probably best to remove the `MetricConfig.cs` file and move the constants into the system itself. The use case for a user-assigned delegate no longer exists